### PR TITLE
meta: upgrade to node v22

### PR DESCRIPTION
**Pull request by ovflowd**
This PR upgrades all references of Node.js v20 to v22 (Vercel already got updated to v22)

### CODE REVIEW

**Comment by ovflowd** [comment]
cc @nodejs/nodejs-website requesting fast track here :)

**Comment by AugustinMauroy** [comment]
-1 for fast track. a fast tack is for modifications that have to be land now if not it degrades the UX

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-20
+v22
```
Comment (AugustinMauroy) [inline .nvmrc:1]: it's work with the `v` prefix ?

**Review by AugustinMauroy — APPROVED** [review]
LGTM !

File: .nvmrc (hunk 1 above)
Comment (bmuenzenmeyer) [reply .nvmrc:1]: yes
https://github.com/nvm-sh/nvmrc/blob/0d325aa903893072cb07daf43ae04b491e104d6c/test/fixtures/valid/basic-comments/.nvmrc

**Comment by ovflowd** [comment]
> -1 for fast track. a fast tack is for modifications that have to be land now if not it degrades the UX

~What sort of bullshit is this, where is this written?~

Apologies for the language, I'm just unsatisfied with these random rules you're coming up out from nowhere. I wouldn't have requested fast-track if this does not apply to a fast-track.

You can still be against the fast-track, but do not use wording that makes it sound that you're basing your decision on the CONTRIBUTING guidelines.

**Comment by ovflowd** [comment]
image

File: apps/site/package.json [hunk 2]
```diff
@@ -1,20 +1,6 @@
 {
-  "type": "module",
-  "private": true,
   "name": "@node-core/website",
-  "description": "Nodejs.org Website",
-  "homepage": "https://nodejs.org",
```
Comment (bmuenzenmeyer) [inline apps/site/package.json:6]: this key supports the [`npm docs` command](https://docs.npmjs.com/cli/v9/commands/npm-docs)

File: apps/site/package.json [hunk 3]
```diff
@@ -1,20 +1,6 @@
...
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/nodejs/nodejs.org.git"
-  },
```
Comment (bmuenzenmeyer) [inline apps/site/package.json:10]: this key supports the [`npm bugs` command](https://docs.npmjs.com/cli/v9/commands/npm-bugs)

Comment (ovflowd) [reply apps/site/package.json:10]: These are already on the root package.json

File: apps/site/package.json (hunk 2 above)
Comment (ovflowd) [reply apps/site/package.json:6]: These are already on the root package.json

**Comment by bmuenzenmeyer** [comment]
yeah i think fast-track can certainly be warranted if we are in a position where our prod deployments in vercel are on a different version than we build and test - that could lead to unexpected behavior. so minimizing the amount of time we are in that position is prudent

**Review by bmuenzenmeyer — APPROVED** [review]

**Comment by ovflowd** [comment]
> yeah i think fast-track can certainly be warranted if we are in a position where our prod deployments in vercel are on a different version than we build and test - that could lead to unexpected behavior. so minimizing the amount of time we are in that position is prudent

My understanding is that fast-tracking is not a matter of urgency, but a matter of complexity.

**Comment by bmuenzenmeyer** [comment]
you are right - urgency is covered earlier in https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md#before-merging

File: .nvmrc (hunk 1 above)
Comment (AugustinMauroy) [reply .nvmrc:1]: Thank

**Pull request merged**